docs/input.md

```diff
@@ -116,6 +117,16 @@ props to be passed to the React Native `Text` component used to display the erro
 
 ---
 
+### `inputComponent`
+
+component that will be rendered in place of the React Native `TextInput` (optional)
+
+|              Type              | Default |
+| :----------------------------: | :-----: |
+| Component (string \| function) |  none   |
```

Comment:
Change this definition here. See https://react-native-training.github.io/react-native-elements/docs/0.19.0/button.html#component for example. The default would also be TextInput